### DISCUSSION TITLE
Apache Superset not sending email and alerts to email and slack

### DISCUSSION THREAD

**Discussion by donmarube** [apache/superset#23134] [Q&A / Help]
I have successfully installed Apache superset 2.0.1 and connected with databases I have created charts and published them to dashboards. I can create the alerts and emails but I have not received any alerts and reports. Kindly help this has scratched my head for 2 weeks now I have tried almost everything

**Comment by sfirke** [comment]
What do you see in the logs for your Superset app and celery worker?

**Reply by donmarube** [reply]
@sfirke > What do you see in the logs for your Superset app and celery worker?

I am using redis, However my celery workers and beat seem to be pointing to the default cache instead of redis. Extra information

: I have definitely installed
 do away with the "No PIL installation found"
I am also running superset using gunicorn (Would have attached the screenshots but they aint loading properly)
![2](https://user-images.githubusercontent.com/43114087/220559737-efd17513-8ab1-47fe-9085-65b082ed27
![3](https://user-images.githubusercontent.com/43114087/220559747-0102a354-0996-4f92-919d-423450b42c12.jpg)
92.jpg)

![1](https://user-images.githubusercontent.com/43114087/220559729-441950e4-b0d7-4055-9b5e-26b54782ed4c.jpg)

**Comment by donmarube** [comment]
images.githubusercontent.com/43114087/220557562-34a04f1f-187e-4c40-9380-098f97eeee04.jpg)

**Comment by sfirke** [comment]
Maybe try `redis` in place of `localhost` in your CeleryConfig?

**Comment by lxycorrine** [comment]
have you figured it out? Got the same problem
